### PR TITLE
Update run-parts

### PR DESCRIPTION
**Pull request by curiosityseeker**

### CODE REVIEW

File: apparmor.d/profiles-m-r/run-parts [hunk 1]
```diff
@@ -15,10 +15,19 @@ profile run-parts @{exec_path} {
...
+  /{usr/,}bin/snapper rix,
+  
+    
   /usr/share/update-notifier/notify-reboot-required   rPx,
   /usr/share/update-notifier/notify-updates-outdated  rPx,
 
   /etc/ r,
+  /etc/conf.d/snapper{,**}                             r,
+  /etc/nsswitch.conf                                   r,
+  /etc/passwd                                          r,
```
Comment (roddhjav) [inline apparmor.d/profiles-m-r/run-parts:29]: Use the nameservice-strict abstraction instead.

**Comment by roddhjav** [comment]
The abstraction file should go in the profile with the other abstraction: `include <abstractions/nameservice-strict>`

Also, for future PR, please do not edit the file directly on Github:
- Write your change locally
- Test the change by building the package, therefore you will catch all compilation issue
- Then commit and push to GitHub
-

**Comment by curiosityseeker** [comment]
I understand and am sorry for the inconveniences. So right now I'm not going to make any further commits. I will rather try to make myself more familiar with github and find a proper workflow for me. I hope that I can be back some time in the future.

**Comment by curiosityseeker** [comment]
Is still anything missing?

**Pull request merged**